service/constants/constants.js

```diff
@@ -54,7 +54,23 @@ const FileConstants = {
     WORD_TYPES: ['doc', 'docx']
 };
 
+const PostsConstants = {
+    JOIN_LEAVE: 'system_join_leave',
+    JOIN_CHANNEL: 'system_join_channel',
+    LEAVE_CHANNEL: 'system_leave_channel',
+    ADD_TO_CHANNEL: 'system_add_to_channel',
+    REMOVE_FROM_CHANNEL: 'system_remove_from_channel',
+    ADD_REMOVE: 'system_add_remove',
+    HEADER_CHANGE: 'system_header_change',
+    DISPLAYNAME_CHANGE: 'system_displayname_change',
+    PURPOSE_CHANGE: 'system_purpose_change',
+    CHANNEL_DELETED: 'system_channel_deleted',
+    EPHEMERAL: 'system_ephemeral'
+};
+
 export default {
     ...Constants,
-    ...FileConstants
+    ...FileConstants,
+    ...PostsConstants,
+    IGNORE_POST_TYPES: [PostsConstants.JOIN_LEAVE, PostsConstants.JOIN_CHANNEL, PostsConstants.LEAVE_CHANNEL, PostsConstants.REMOVE_FROM_CHANNEL, PostsConstants.ADD_TO_CHANNEL, PostsConstants.ADD_REMOVE]
```

Comment:
I can see that the order of the ignored types here differs from the order of the general `PostsConstants`. It hindered the ability to quickly determine which post types are not in the ignored list.
This is an indicator we might wanna start embracing alphabetical ordering for flat, static arrays & objects like this. It will eliminate some of the inconsistencies that add to our cognitive load. 
cc @hmhealey @jarredwitt